contents/handbook/growth/sales/new-sales.md

```diff
@@ -64,31 +64,47 @@ This is basically a combo of discovery and demo call - your objectives here are
 - Find out as much as you reasonably can about them
 - Have a solid plan for next steps in place
 
-We have a simple deck (seriously, it's like 5 slides) - ask someone on the Sales team for an invite to our Pitch account so you can create customized versions with your name etc. 
+We have a simple deck (seriously, it's like 5 slides) - ask someone on the Sales team for an invite to our Pitch account so you can create customized versions with your name etc.
 
 You should give a relevant and pointed demo - don't just throw everything in, as the customer will get overwhelmed. If you don't show what's important first, typically more senior people on the call will become distracted.
 
 For example, a customer may say "we need to see how our customers our using our platform". In this case, a good approach is to go straight to Session Replay, then tie Replay into Analytics, then go from there.
 
 > Start with what their biggest problem/request is, stay there until they are happy, then move on to point two. We don't want to fall into the trap of doing the same demo for each customer regardless of what they say at the beginning.
 
-Make sure you cover:
+#### Exit criteria
 
-- Who is there and what their roles are - in particular, are they the decision-makers at their company?
-- Why do they _need_ PostHog?
-- Opportunity for them to ask up-front questions and get direct answers to them
+Before moving on to an evaluation, we should know the following (inspired by [BANT](https://www.scratchpad.com/blog/bant) and [MEDDPICC](https://www.scratchpad.com/blog/meddpicc)):
+
+* Pain - Do they have a problem?  Is it painful enough right now that they are willing to adopt a new solution to solve it?  Will PostHog solve this problem for them?
+* Pricing - We should know the rough size of the opportunity, and that that is in line with expectations and budget of the customer.
+* Decision - How are they evaluating us?  Is it competitive?  Who?  What is their criteria for success?
+* Champion - Are you working with a champion who is going to sell internally?  Are they the buyer or do they know who the buyer is?
+* Paper process - After a successful evaluation what happens next?  Do they need a Custom DPA/BAA/MSA?  Is there a security review needed?  Who signs the Order Form?
+* Timeline - When do whey want to have a contract signed/solution in place?
+
+These should all be documented in the Salesforce Opportunity record before moving to Product Evaluation.  Note that you shouldn't ask these as a set of interrogation questions (or even worse email them to the customer ahead of time) but can work them conversationally into your demo meeting flow as follows:
```

Comment:
I think these are the right questions - do you think that it should be mandatory in Salesforce to have answers to these filled out in some kind of form before you can move an opportunity to a product evaluation? 

The way you've phrased these, I think it is reasonable to be strict here, but I'm curious if the team agrees or will we end up with BS-y stuff added to Salesforce.

Reply:
I think:
1. We should have separate text fields in Salesforce for these
2. We shouldn't yet make them mandatory but should use gaps to drive our discussions about the deal, as otherwise we risk garbage going in there just to make an opp move forward

Reply:
Agreed on making separate fields, but not making fields mandatory yet. Too much process until we've tested it and figured out this is the right approach would be a hinderance. Better to reinforce during our Pipeline reviews. Also, some deals are just funky and may require a slightly different process.

Reply:
I agree with making them not mandatory for now. Effective and sensible criteria like these do take a sizeabale amount of time and effort to fine tune. 

I subscribe to the implemat, experiment and iterate idea. What works for other sales orgs might and will not work for us. 

Thus, not mandatory fields for now, with an active effort to fill those out nonetheless and revisit after a few runs to see what needs to change.